src/Components/InteractiveIframe/iframeCard.jsx

```diff
@@ -36,8 +44,12 @@ export default class IframeCard extends Component {
 
   subscribeToApplicationEvents = () => {
     const em = this.props.eventManager;
-    const { url } = this.props;
     const { eventIds } = this.props;
+    const { params } = this.props;
+    let { url } = this.props;
+    if (params) {
+      url = URLUtils.template(url, params);
```

Comment:
URLUtils should be proper camel case. UrlUtils, easier to read. same for the file name